lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -2455,19 +2464,22 @@ balanceTransactionSpec = describe "balanceTransaction" $ do
         it "roundtrips with toCardanoValue" $
             property prop_posAndNegFromCardanoValueRoundtrip
 
-    describe "effect of txMaxSize on coin selection" $ do
-        let addr = Address $ unsafeFromHex
-                "60b1e5e0fb74c86c801f646841e07\
-                \cdb42df8b82ef3ce4e57cb5412e77"
-        let mw = SomeMnemonic $ either (error . show) id
-                (entropyToMnemonic @12 <$> mkEntropy "0000000000000000")
-        let rootK = Shelley.unsafeGenerateKeyFromSeed (mw, Nothing) mempty
+    it "increases zero-ada outputs to minimum" $ do
+        let era = WriteTx.RecentEraBabbage
+        let out = TxOut dummyAddr (TokenBundle.fromCoin (Coin 0))
```

Comment:
```suggestion
        let out = TxOut dummyAddr mempty
```

Reply:
I think `(TokenBundle.fromCoin (Coin 0))` makes more sense over `mempty` here for the sake of visually highlighting the difference to `out'`:

```haskell
let out = TxOut dummyAddr (TokenBundle.fromCoin (Coin 0))
let out' = TxOut dummyAddr (TokenBundle.fromCoin (Coin 874_930))
```